charts/replicated-library/templates/lib/_collector_templates.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "replicated-library.troubleshoot.collector.template.cluster" -}}
+- {{ $ }}: {}
+{{- end }}
+
```

Comment:
I assume this is the clusterInfo template, let's put the same name as the actual collector just for clarity.

Reply:
changed